avaframe/com1DFAPy/com1DFACfg.ini

```diff
@@ -17,6 +17,10 @@ rhoEnt = 200
 massPerPart = 2000
 # minimum flow depth [m]
 hmin = 0.05
+
+#+++++++++++++SPH parameters
+# SPH gradient option
+sphOption = 2
```

Comment:
I think the ini files should contain a quick description +  the available options